File: tests/fixtures/maven-mini/src/test/java/com/example/PaymentRulesTest.java
package com.example;

import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

class PaymentRulesTest {
    @Spec("maven java scenario")
    @Test
    void approvesValidCard() {
        assertTrue("424242".startsWith("42"));
    }
}

File: tests/fixtures/gradle-kotlin-mini/src/test/kotlin/com/example/PaymentRulesTest.kt
package com.example

import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test

class PaymentRulesTest {
    @Spec("gradle kotlin scenario")
    @Test
    fun approvesValidPayment() {
        assertTrue("paid".startsWith("pa"))
    }
}
